doc/ref_arch/kubernetes/chapters/chapter04.md

```diff
@@ -30,10 +52,48 @@
 > * Which optional features are used and which optional API-s are available
 > * Which [alfa or beta features](https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/) are used
 
+The distribution and version of Kubernetes used must be listed in the [Kubernetes Distributions and Platforms document](https://docs.google.com/spreadsheets/d/1LxSqBzjOxfGx3cmtZ4EbB_BGCxT_wlxW_xgHVVa23es/edit#gid=0) which lists all those that are Certified as being conformant with the CNCF Kubernetes specification.  In order to align with the [Kubernetes version support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-versions), a Reference Implementation must use one of three latest minor versions (`n-2`) - e.g. if the latest version is 1.17 then the RI must use either 1.17, 1.16 or 1.15.
+
+This Reference Architecture also specifies:
+
+- Master nodes must run the following Kubernetes control plane services:
+    - kube-apiserver
+    - kube-scheduler
+    - kube-controller-manager
+- Master nodes can also run the etcd service and host the etcd database, however etcd can also be hosted on separate nodes
+- In order to support `req.gen.rsl.01`, `req.gen.rsl.02` and `req.gen.avl.01` a Reference Implementation must:
+    - Consist of either three, five or seven nodes running the etcd service (can be colocated on the master nodes, or can run on separate nodes, but not on worker nodes)
+    - At least one master node per availability zone or fault domain to ensure the high availability and resilience of Kubernetes control plane services
+    - At least one worker node per availability zone or fault domain to ensure the high availability and resilience of workloads managed by Kubernetes
```

Comment:
"At least one worker node per availability zone or fault domain to ensure the high availability and resilience of workloads managed by Kubernetes"

As stated it seems to imply that we only need at least one worker node in each AZ/fault domain for workload resiliency.  This needs rewording for workload resiliency: "Workload resiliency requires the need to run them on at least one worker node per availability zone or fault domain."

Reply:
> As stated it seems to imply that we only need at least one worker node in each AZ/fault domain for workload resiliency.

That is correct, and I'm not sure how your suggestion differs from the existing wording?  If I was to move "Consist of" from the first bullet to the statement before the list, would that make more sense?

Reply:
@tomkivlin Maybe the sentence needs improving but I tried to state that the workload must be "run" on worker nodes in each AZ: "Workload ... need to run them on at least one worker node per availability zone" -- if not clear then does the following reword help: "Workload resiliency requires the need to run the workload on at least one worker node per availability zone or fault domain."

Reply:
Ah I see - I think that is outside the scope of this RA. The requirement is to support workload resilience (which having one worker node in each AZ does), not to specify how/where the workloads are run.  Bit nit-picky I know, but I am keen to avoid too much specification that isn't about the platform.  Is that ok?